src/index.js

```diff
@@ -1,100 +1,138 @@
-import { initializeService } from './services/config.js';
+/*** This file was generated automatically. To recreate, please run `npm run build-index`. ***/
 
 import {
-  fetchSongById,
-  fetchArtists,
-  fetchSongArtistCount,
-  fetchRelatedSongs,
-  fetchAllSongs,
-  fetchSongFilterOptions,
-  fetchSongCount,
-  fetchWorkouts,
-  fetchNewReleases,
-  fetchUpcomingEvents,
-  fetchByRailContentId,
-  fetchByRailContentIds,
-  fetchAll,
-  fetchAllFilterOptions,
-  fetchMethodNextLesson,
-  fetchFoundation,
-  fetchMethod,
-  fetchMethods,
-  fetchMethodChildren,
-  fetchMethodChildrenIds,
-  fetchNextPreviousLesson,
-  fetchRelatedLessons,
-  fetchAllPacks,
-  fetchPackAll,
-  fetchPackChildren,
-  fetchLessonContent,
-  fetchCourseOverview,
-  fetchChildren,
-  fetchParentByRailContentId,
-  fetchMethodPreviousNextLesson,
-  fetchLiveEvent,
-  fetchChallengeOverview,
-  fetchCoachLessons,
-  fetchByReference,
-  fetchScheduledReleases,
-  fetchArtistLessons,
-  fetchGenreLessons,
-  fetchRelatedMethodLessons,
-} from './services/sanity.js';
```

Comment:
This adds entries that are exported in sanity.js but were missing before:

- fetchCatalogMetadata,
- fetchSanity,
- getSortOrder